zamba/models/cnnensemble/src/prepare_train_data.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import StratifiedKFold
+import random
+import os
+from PIL import Image
+from multiprocessing import Pool
+from tqdm import tqdm
+
+from zamba.models.cnnensemble.src import config
+from zamba.models.cnnensemble.src import utils
+
+
+def generate_folds(config):
```

Comment:
If we pass a config, it should have a different name so it doesn't conflict with the `... import config` above.